### ISSUE TITLE
Unable to check GraphQL query selection boxes

### ISSUE THREAD

**Issue by JimmyRice** [a85/POSTMan-Chrome-Extension#11777]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues and I know that duplicates will be closed

### Describe the Issue

After selecting a field in Postman, the query statement is generated on the right-hand side, but the selection box is automatically unchecked. This happens not only with fields, but also with parameter selection boxes.

This results in the inability to generate multiple queries or select multiple fields.

### Steps To Reproduce

1. Open Postman.
2. Create a new GraphQL query.
3. Use the URL address: https://graphql.anilist.co/ (or any other GraphQL URL).
4. Try to select a field.

### Screenshots or Videos

![PostmanGif](https://i.imgur.com/LufbPlK.gif)

### Operating System

Windows

### Postman Version

10.11.1

### Postman Platform

Postman App

### User Account Type

Signed In User

**Comment by SterlingChin** [comment]
I am unable to reproduce this issue. I'm on `v10.11.1` and used the URL that you provided. I was able to create and execute a query.  I also tested it in the browser and everything worked as expected

Some things to try:
* Try the GraphQL client in the browser and see if you can run a query
* Try clearing the cache and reloading the app
* Is there a VPN or AdBlocker that is enabled that might be interfering?

The only difference that I can see is that I'm on a Mac and not on a Windows computer. If you are still having problems with the new client after trying this, we can hop on a call to debug further.

**Comment by JimmyRice** [comment]
Hi, thank you for your reply.

I tried clearing the cache and reloading, but the result is still the same.

Maybe I can try using my Mac later.

**Comment by JimmyRice** [comment]
Yes, you are correct. The Mac version of Postman does not seem to have encountered the same issue.

In Windows, I have tried clearing the cache and restarting the Postman client, as well as trying the web version of Postman (using Edge, Chrome, and Firefox browsers).

I also tested it on other people's computers, and they have the same issue with Postman.

Our common factor is that we are using Windows 11 as our operating system, with version number 22H2 and OS Build 22621.1265.

**Comment by SterlingChin** [comment]
Thank you for the additional context and information. We're tracking this issue and actively working on fixing the bug.

**Comment by chrisstone** [comment]
Having the same problem. Windows 10 22H2, Postman v10.11.1 desktop app.

**Comment by capooods** [comment]
Also having the same issue. 

Version 10.11.1
UI Version 10.11.1-ui-230222-1424
Desktop Platform Version 10.11.1
Architecture x64
OS Platform win32 10.0.22621

**Comment by SterlingChin** [comment]
A fix for this issue has been merged and will be part of Postman `v10.12` and will start to roll out early next week.

**Comment by JimmyRice** [comment]
Awesome! Thank you very much for your prompt response. Can you share the reason for the issue? I am very curious why Postman behaves differently on Mac and Windows. XDDD

**Comment by SterlingChin** [comment]
Great question.

TLDR: Line Endings. Part of the parse step for figuring out the query from the editor didn't handle Windows line endings right (Windows `\r\n` vs. Mac `\n`).

**Comment by JimmyRice** [comment]
WOW

So that's how it is, everything makes sense now. Thank you for fixing it so quickly.
